packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -506,7 +506,7 @@ QUnit.module('Tab select action', () => {
     });
 
     QUnit.test('focusedElement must be changed after changing the selectedIndex', function(assert) {
-        assert.expect(2);
+        assert.expect(3);
```

Comment:
This code is not asyncroniously. All of your assertion will always execute . I suggest there is no need for this line.

Reply:
[25e09d20c8724be348f469868a7e2f46a2e55633]